### ISSUE TITLE
Can't run docker commands in container 

### ISSUE THREAD

**Issue by Mbarzegar-unimelb** [remicres/otbtf#42]
Hello

I'm trying to follow the tutorial but I can't run any docker command in the container. I need to connect my host directory to the container to be able to use the .tif images.  

error= bash: docker: command not found

I can't use sudo because it asks for a password.

**Comment by Pratyush1991** [comment]
I think when you start the container you have to mount the host volume. Then you can access it I guess.
Let say the host dir folder that you want to mount inside the container is: `/home/abc/images`

You can start the docker container (Normal Case):
`sudo docker run -ti  --rm --gpus all -u otbuser -v /home/abc/images:/home/otbuser mdl4eo/otbtf2.0:gpu bash`

If you want to enter the container as 'root' then pass the user option as root:
`sudo docker run -ti  --rm --gpus all -u root -v /home/abc/images:/home/otbuser mdl4eo/otbtf2.0:gpu bash`

**Comment by Mbarzegar-unimelb** [comment]
I ran the command "sudo docker run -ti --rm --gpus all -u root -v /home/abc/images:/home/otbuser mdl4eo/otbtf2.0:gpu bash" and changed it based on my host directory and it started downloading the package again. Then I tried to enter to the container as root user using below command
docker run -u 0 -it mdl4eo/otbtf2.0:cpu bash
I tried to run the commands in tutorial (I already put the image in my host folder) and I got the below error:

(FATAL) LabelImageSampleSelection: Cannot open image /root/home/maryam/terrain_truth_epsg32654_A.tif. The file does not exist.

Am I doing something wrong? I just started using docker
